docs/en/connector-v2/source/Paimon.md

```diff
@@ -40,6 +41,10 @@ The table you want to access
 
 The file path of `hdfs-site.xml`
 
+### projection [string]
```

Comment:
```suggestion
### read_columns [string]
```
How about like other connectors, named `read_columns` https://seatunnel.apache.org/docs/connector-v2/source/LocalFile#read_columns-list

Reply:
@Hisoka-X @hailin0 I have fixed it as suggested, please review it again.